CI_YAML.md

```diff
@@ -168,3 +168,14 @@ Cocoon supports tests that are not owned by Flutter infrastructure. By default,
      scheduler: my_external_location
    ```
 8. Send updates to `https://flutter-dashboard.appspot.com/api/update-task-status` - https://github.com/flutter/cocoon/blob/master/app_dart/lib/src/request_handlers/update_task_status.dart
+
+
+## Scheduling Targets
```

Comment:
Can we add subsections for pre-submit and post-submit, as they are both scheduled via cocoon now.

Reply:
Done